src/main/java/net/dv8tion/jda/internal/entities/AbstractMessage.java

```diff
@@ -480,34 +481,39 @@ public boolean isPinned()
         return null;
     }
 
+    @NotNull
```

Comment:
This is the wrong annotation, we use javax.annotation only (as you can see in the imports, this is a different one)

Reply:
Oh whoops, sorry about that!